README.md

```diff
@@ -3,8 +3,257 @@ Nexus
 Nexus is a scalable and efficient serving system for DNN applications on GPU
 cluster.
 
-Contents
---------
-* [Compile from source](docs/compile.md)
-* [Build docker image](docs/docker.md)
-* [Deploy Nexus](docs/deploy.md)
+Compile and Deploy Nexus
+========================
+
+Download nexus and sample models
+--------------
+
+### Download Nexus
+ 
+Download nexus and framworks it supports from github
+```
+$ git clone --recursive https://github.com/uwsaml/nexus.git
+$ cd nexus
+$ git submodule update --init --recursive
+```
+
+### Download Model Zoo
+
+Nexus publishes public model zoo on the Amazon S3. To download the model zoo
+from S3, you need to install [AWS CLI](https://aws.amazon.com/cli/) via
+`pip install awscli`, and configure AWS CLI by `aws configure`.
+
+The configuration will prompt you to provide your AWS access key ID and
+secret access key. Instructions for creating the access key pair can be found
+at [AWS user guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+To download the Nexus public model zoo, run
+```
+$ mkdir nexus-models
+$ aws s3 sync s3://nexus-models nexus-models/
+```
+
+Pre-requisites
+--------------
+
+Recommended development environment on Ubuntu (>=16.04)
+
+### General dependencies
+```
+$ [sudo] apt-get install libboost-all-dev libgflags-dev libgoogle-glog-dev \
+libgtest-dev libyaml-cpp-dev libopenblas-dev libleveldb-dev liblmdb-dev \
+libhdf5-serial-dev
+```
+
+### Required libraries
+* protobuf >= 3.5.0
+* [grpc](https://github.com/grpc/grpc/blob/master/INSTALL.md) >= v1.4.x
+* OpenCV >= 3.0
+* bazel >= 0.10.0
+* CUDA >= 8.0
+* CUDNN >= 6.0
+
+### Pre-requsites for Deploy Nexus Service
+
+Nexus is able to be compiled from source, but if it is necessary to run Nexus service across multiple servers, we use the use docker swarm. So we recommand to deploy nexus by docker.
+
+To build nexus docker images and deploy Nexus, following packages are required
+* [CUDA 8.0](https://developer.nvidia.com/cuda-80-ga2-download-archive)
+* [docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/)(>=1.12)
+* [nvidia-docker](https://github.com/NVIDIA/nvidia-docker)
+
+Set Config for Nexus
+--------------------
+
+There is a file names Makefile.config in the nexus folder. And it stores location of cuda and cudnn. $(CUDA_PATH) should be set as your/path/to/cuda, and $(CUDNN_PATH) should be set as your/path/to/cudnn. 
```

Comment:
"And it stores location of cuda and cudnn." --> "You can config the CUDA and CUDNN library path in the config file"
change "your/path/to/cuda" to "CUDA library path", change "your/path/to/cudnn" to "CUDNN library path".